Makefile

```diff
@@ -47,6 +47,16 @@ container:
     --build-arg REPO_INFO=${REPO_INFO} \
     -t ${IMAGE}:${TAG} .
 
+.PHONY: multi-arch
+multi-arch:
+	docker buildx build \
```

Comment:
I'm not familiar with docker. I've `19.03.12` installed and `docker buildx` is not a sub-command in my installation.

Reply:
Good call, looks like this part is pretty easy as of Docker 19.03. https://github.com/Kong/kubernetes-ingress-controller/blob/8132400700df912278e2d9d2592f3c1a54648de4/.ci/build-main.sh#L6 

https://docs.docker.com/buildx/working-with-buildx/